spec/unit/views/components/table_for_spec.rb

```diff
@@ -301,6 +301,46 @@
       end
     end
 
+    context 'when i18n option is specified' do
+      before(:each) do
+        I18n.backend.store_translations(:en, {
+          activerecord: { attributes: { post: { title: "Name" } } }
+        })
+      end
+
+      let(:table) do
+        render_arbre_component assigns, helpers do
+          table_for(collection, i18n: Post) do
+            column :title
+          end
+        end
+      end
+
+      it "should use localized column key" do
+        expect(table.find_by_tag("th").first.content).to eq "Name"
+      end
+    end
+
+    context 'when i18n option is not specified' do
+      before(:each) do
+        I18n.backend.store_translations(:en, {
+          activerecord: { attributes: { post: { title: "Name" } } }
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.